src/libs/actions/Report.js

```diff
@@ -543,16 +537,16 @@ function fetchActions(reportID, offset) {
 function fetchAll(shouldRedirectToReport = true, shouldRecordHomePageTiming = false) {
     fetchChatReports()
         .then((reportIDs) => {
-            if (shouldRedirectToReport && !currentURL.includes(ROUTES.REPORT)) {
-                // Redirect to either the last viewed report ID or the first report ID from our report collection
+            if (shouldRedirectToReport) {
+                // Update currentlyViewedReportID to be our first reportID from our report collection if we don't have
+                // one already.
                 if (lastViewedReportID) {
-                    Onyx.set(ONYXKEYS.CURRENTLY_VIEWED_REPORTID, String(lastViewedReportID));
-                } else {
-                    const firstReportID = _.first(reportIDs);
-                    if (firstReportID) {
-                        Onyx.set(ONYXKEYS.CURRENTLY_VIEWED_REPORTID, String(firstReportID));
-                    }
+                    return;
                 }
+
+                const firstReportID = _.first(reportIDs);
+                const currentReportID = firstReportID ? String(firstReportID) : 0;
```

Comment:
Should be `''` instead of `0` to maintain consistency with our reportID types?

Also got this error when testing on a new account: 
![image](https://user-images.githubusercontent.com/16747822/111387939-c33b5400-866b-11eb-965e-e4f9d435210b.png)

Reply:
Nice catch! I'll fix this!